network/composition.yaml

```diff
@@ -19,9 +25,16 @@ spec:
             cidrBlock: 192.168.0.0/16
             enableDnsSupport: true
             enableDnsHostNames: true
+            tags:
+            - key: Env
```

Comment:
this env: dev tag could be a bit misleading - i'm wondering if its the best example to bake something into the composition that is dependent on the env it runs in.  e.g. what if this composition is installed into the staging control plane instead?  maybe there's a better pattern for that, and we shouldn't be showing tag usage in compositions for env dependent details like this?

Reply:
Good catch! This is copy-paste from one of my networking examples. Indeed version tag should come from composition rather than be hardcoded. I will remove it and leave the Owner tag because it is platform-related and will be independent of the environment and will help find resources in the cloud belonging to the Platform Team.
The name tag could also be transformed similarly to the other ones by suffixing a unique name from the claim

Reply:
Removing the env dev and leaving, for now, name as it is, because it runs only one example of a network. Something to think about in future releases.